docs/admin_guide.md

```diff
@@ -421,6 +426,15 @@ npm run build # Builds frontend
 
 Make sure you have substituted all the passwords and configured the basic services.
 
+#### Let's Encrypt (TLS)
+
+Follow the prompts to create TLS certificates; if you didn't already have nginx up, you can use
+standalone mode.
+
+```bash
+sudo certbot certonly
+```
```

Comment:
missing instructions? like maybe you probably need to specify `-d mbin.domain.tld` to specify the domain(s) to request certs for or something

(I can't help you fill in this as I haven't used certbot, If anyone has more info and could help fill this in that'd be appreciated)

Reply:
'certbot certonly' gives prompts asking for anything that's not specified; my feeling was that it made more sense to let the tool just prompt people for info it needed, as opposed to giving them a command line they need to figure out the meaning of and then modify. Unless there are cases where this needs to be able to run unattended based on these instructions or etc?

Reply:
ah ok, looks like I missed the "Follow the prompts" up there, all right.
also it doesn't look like there's need/support for installing bare metal unattended at the moment.

---

my experience in this regard was from `acme.sh` corner where I specify all needed arguments up front, which can look something like this:

```
% acme.sh \
  --install-cert \
  -d domain.tld -d '*.domain.tld' \
  --dns dns_cf \
  --fullchain-file /usr/local/etc/nginx/ssl/wildcard_domain.tld.crt \
  --key-file /usr/local/etc/nginx/ssl/wildcard_domain.tld.key \
  --reloadcmd 'service nginx forcereload'
```

so when I saw/compared the new command with the old command(s) I got kinda confused and thought if I have to go look up the `certbot` arguments and have to fill it myself or something.